kasa/tests/fixtures/HS100(UK)_4.1_1.1.0.json

```diff
@@ -1,43 +1,46 @@
 {
-    "emeter": {
-        "err_code": -1,
-        "err_msg": "module not support"
-    },
-    "smartlife.iot.common.emeter": {
-        "err_code": -1,
-        "err_msg": "module not support"
-    },
-    "smartlife.iot.dimmer": {
-        "err_code": -1,
-        "err_msg": "module not support"
-    },
-    "smartlife.iot.smartbulb.lightingservice": {
```

Comment:
We are going to lose the information about unsupported modules completely now, I'm wondering if we should try to keep this available for tests, though?

Reply:
That’s just what the dump_devinfo logic gave me. I’m not sure why. Any ideas? (The first time I uploaded this fixture I edited another one because the tool didn’t work)

Reply:
Ohh, hmm. Maybe it has something to do with modules not being fetched during the query that lands into the fixture file? Basically, the initial update sees that the module is not supported so it won't be fetched again in the follow up queries...

Mmh, I think it's fine though as we have tests in place for the unsupported modules and their query behavior. Maybe the only reason to keep these around would be to make it easy to see which modules are supported by specific model, but it's not that we are currently even testing against all of them on all devices...